packages/design-system/src/components/Table/Table.jsx

```diff
@@ -143,10 +143,7 @@ export class Table extends React.PureComponent {
 
 Table.defaultProps = {
```

Comment:
We should be adding a `dense` prop to the table just like `striped`

Reply:
Also make sure to update the Table definition files, I think those are probably outdated and need to be tested